SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialAuthorityTest/SpatialAuthorityTest.cpp

```diff
@@ -468,6 +499,117 @@ void ASpatialAuthorityTest::PrepareTest()
 	}
 }
 
+void ASpatialAuthorityTest::CheckDoesNotMigrate(ASpatialAuthorityTestActor* Actor, int ServerId)
+{
+	const FWorkerDefinition& LocalWorkerDefinition = GetLocalFlowController()->WorkerDefinition;
+	if (LocalWorkerDefinition.Type == ESpatialFunctionalTestWorkerType::Server)
+	{
+		// Allow it to continue working in Native / Single worker setups.
+		if (GetNumberOfServerWorkers() > 1)
+		{
+			if (LocalWorkerDefinition.Type == ESpatialFunctionalTestWorkerType::Server && LocalWorkerDefinition.Id == ServerId)
+			{
+				if (VerifyTestActor(Actor, ESpatialHasAuthority::ServerAuth, ServerId, ServerId, 1, 0))
```

Comment:
Bit of a driveby - but it seems like VerifyTestActor should also validate `ReplicatedAuthWorkerIdOnBeginPlay`
Perhaps worth creating a follow-up for the additional validation.

Reply:
Yes I can do this as a follow-up task.